components/bear-market-airdrop/Blocks/Hero/Unboxed.tsx

```diff
@@ -0,0 +1,295 @@
+/* eslint-disable no-restricted-imports */
+
+/* eslint-disable react/forbid-dom-props */
```

Comment:
don't disable this it's there for a reason. to apply styling to a span you can do
```jsx
import {Box} from "@chakra-ui/react";
<Box as="span" {...normal chakra styling></Box>
```